app/sidekiq/event_bus_gateway/va_notify_email_status_callback.rb

```diff
@@ -7,6 +7,22 @@ def self.call(notification)
 
       add_metrics(status)
       add_log(notification) if notification.status != 'delivered'
+      if notification.status == 'temporary-failure' && Flipper.enabled?(:event_bus_gateway_retry_emails)
+        retry_email(notification)
+      end
```

Comment:
Is excessive retries a concern? Can we somehow set a maximum number of retries?

Reply:
yeah that's a good question. I think it's harmless to introduce a retry count -- though notably this isn't a literal redis retry, I am re-queueing the job -- but this strikes me as a VA Notify question. I'll ask.

Reply:
Yes, they did not have a hard-and-fast recommendation but suggested we find an existing example within vets-api and maintain that convention. Dariusz saw one (and perhaps the only such) example, which uses redis's built-in retries (it queues a "status update" job and that job raises an exception if the status from VA Notify is a failure).